packages/slider/sp-slider.ts

```diff
@@ -9,6 +9,7 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
+import './sp-slider-handle'; // codify sp-slider's implicit dependency on sp-slider-handle
```

Comment:
I thought we'd added linting to help prevent this, but apparently not. We want to use file extensions on our imports, so `.js`, here.

We use export maps, now, so this won't break in almost all cases, but it's safer to include them.

Reply:
Good catch!